.github/workflows/renovate-checks.yml

```diff
@@ -0,0 +1,80 @@
+name: Renovate Checks
+on:
+  pull_request:
+    branches:
+      - renovate/**
+      - test-all/renovate/**
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Git mail
+        run: git config --global user.email "kitt@amazee.com"
+
+      - name: Git username
+        run: git config --global user.name "K.I.T.T."
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          # We can upgrade to PHP 8 when
+          # - https://github.com/mglaman/phpstan-drupal supports it
+          php-version: '7.4'
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get Yarn cache directory
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Get Yarn version hash
+        id: yarn-version
+        run: echo "::set-output name=hash::$(yarn --version | shasum | cut -d' ' -f1)"
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer global config cache-files-dir)"
+      - name: Get Composer version hash
+        id: composer-version
+        run: |
+          echo "::set-output name=hash::$(composer --version | shasum | cut -d' ' -f1)"
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-dependencies
+        with:
+          path: |
+            ${{ steps.yarn-cache.outputs.dir }}
+            ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ steps.yarn-version.outputs.hash }}-composer-${{ steps.composer-version.outputs.hash }}-github_run_id-${{ github.run_id }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-${{ steps.yarn-version.outputs.hash }}-composer-${{ steps.composer-version.outputs.hash }}-
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Check if there are config changes after Drupal updates
+        run: |
+          set -e
+          cd apps/silverback-drupal
+          source .envrc
+          composer i
+          silverback setup --no-config-import
+          drush cex -y
+          cd -
+          if [[ $(git status --porcelain -- apps/silverback-drupal/config) ]]
+          then
+            >&2 echo 'Error: Found uncommitted Drupal config. Please checkout this branch locally, run `silverback setup --no-config-import && drush cex -y` and commit the config changes.'
```

Comment:
I think this solution is perfect for now. Just out of curiosity: does an auto-commit not work or did you just not invest time there?

Reply:
Yes, there is no rush. The manual resolution is also fine. And if it happens often, we can automate it.

Reply:
I'd leave it manual forever:
- config updates are rare case (I think 😅)
- if they happen, it might make sense to review the changes